gtk/src/default/gtk-3.20/_drawing.scss

```diff
@@ -193,21 +193,22 @@
   // normal button
   //
     color: $tc;
-    outline-color: $focus_border_color;
+    outline-color: $focus_border_color; // Yaru change: we detached focus from selection
     border-color: if($c != $bg_color, _border_color($c), $borders_color);
     // border-bottom-color: if($c != $bg_color, _border_color($c, true), $alt_borders_color);
+    // Yaru change: we don't want to use gradients on buttons
     $button_fill: if($c == $bg_color, if($variant == 'light', image(lighten($c, 2%)), image(lighten($c, 4%))), image(lighten($c, 2%))) !global;
     background-image: $button_fill;
     @include _button_text_shadow($tc, $c);
-    @include _shadows(0 1px transparentize(black, 0.9));
+    @include _shadows(0 1px transparentize(black, 0.9)); // Yaru change: paper effect shadow
```

Comment:
I am afraid that using this terminology the comment is not clear. What do you mean by "paper effect shadow"?

Reply:
Okay let's use something else - what would you suggest? Yaru buttons?

Reply:
see comment above